pkg/operator/ceph/cluster/osd/spec.go

```diff
@@ -1166,6 +1167,32 @@ func (c *Cluster) getExpandPVCInitContainer(osdProps osdProperties, osdID string
 	}
 }
 
+func (c *Cluster) getExpandInitContainer(osdProps osdProperties, configDir, namespace, osdID string, osdInfo OSDInfo) v1.Container {
+	// Output example is in the comments at the beginning of `getExpandPVCInitContainer`().
+	osdDataPath := activateOSDMountPath + osdID
+
+	_, adminKeyringVolMount := cephkey.Volume().Admin(), cephkey.VolumeMount().Admin()
+	volMounts := []v1.VolumeMount{
+		{Name: activateOSDVolumeName, MountPath: osdDataPath},
+		{Name: "devices", MountPath: "/dev"},
+		{Name: k8sutil.ConfigOverrideName, ReadOnly: true, MountPath: tempEtcCephDir},
```

Comment:
Are this volume and `adminKeyVolMount` and  neccessary in this container? I guess `bluefs-bdev-expand` works without these volumes. CMIIW.

Reply:
You are right. They are not neccessary. I fixed.